CHANGELOG.md

```diff
@@ -1,5 +1,427 @@
 # Changelog
 
+## [1.13.0] (2024-07-09)
+
+[1.13.0]: https://github.com/crystal-lang/crystal/releases/1.13.0
+
+### Features
+
+#### lang
+
+- Allow rescuing exceptions that include a module ([#14553], thanks @Blacksmoke16)
+- *(macros)* Allow assignment to `_` inside macro expressions ([#14452], thanks @HertzDevil)
+
+[#14553]: https://github.com/crystal-lang/crystal/pull/14553
+[#14452]: https://github.com/crystal-lang/crystal/pull/14452
+
+#### stdlib
+
+- *(collection)* Add `Array#insert_all` ([#14486], thanks @summer-alice)
+- *(collection)* Improve compile time error for `#sort(&block : T, T -> U)` ([#14693], thanks @beta-ziliani)
+- *(concurrency)* Add `WaitGroup` synchronization primitive ([#14167], thanks @ysbaddaden)
+- *(concurrency)* Add `Thread.new` yields itself ([#14543], thanks @ysbaddaden)
+- *(concurrency)* Add support for `Atomic(Bool)` ([#14532], thanks @ysbaddaden)
+- *(concurrency)* Add `Thread.sleep(Time::Span)` ([#14715], thanks @ysbaddaden)
+- *(files)* **[deprecation]** Move `File.readable?`, `.writable?`, `.executable?` to `File::Info` ([#14484], thanks @straight-shoota)
+- *(files)* Implement `IO#tty?` in Win32 ([#14421], thanks @HertzDevil)
+- *(files)* Implement `File.readable?` and `.writable?` in Win32 ([#14420], thanks @HertzDevil)
+- *(files)* Make `File.readable?` and `.writable?` follow symlinks on Windows ([#14514], thanks @HertzDevil)
+- *(llvm)* Add some missing `LLVM::Context` bindings ([#14612], thanks @ysbaddaden)
+- *(llvm)* Do not strip the macOS target triple ([#14466], thanks @hovsater)
```

Comment:
Maybe a bugfix?

Reply:
Depends on perspective. I understand it as a feature enhancement of the LLVM library to include the deployment version into the default target triple.